src/application/application.go

```diff
@@ -48,10 +49,12 @@ func StartApplication() {
 			likerepository.NewLikeRepository(database),
 			dislikerepository.NewDislikeRepository(database),
 			commentRepository.NewCommentRepository(database),
+			media_grpc_client.NewMediaGrpcClient(),
```

Comment:
Declare variables for each service and repository since it could be used by other components later